### PR TITLE
Simplify the files included in kube-config in snapcraft.yaml

### PR DESCRIPTION
**Pull request by dholbach**
cc @stealthybox

### CODE REVIEW

**Comment by dholbach** [comment]
I pinged Leigh, as this was relevant to some of his work, but can be reviewed by anyone really.

**Comment by stealthybox** [comment]
Does this allow recursive access to any kubeconfig file in the `~/.kube` directory?
If so, +++++++100

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -58,10 +58,8 @@ plugs:
   kube-config:
     interface: personal-files
     read:
-    - $HOME/.kube/config
-    - $HOME/.minikube/client.key
-    - $HOME/.minikube/client.crt
-    - $HOME/.minikube/ca.crt
+    - $HOME/.kube
+    - $HOME/.minikube
```
Comment (stealthybox) [inline snap/snapcraft.yaml]: To support `k3d` we would need access to the host's `$XDG_CONFIG_HOME`, but that does not appear to be supported well by `snap`: [thread](https://forum.snapcraft.io/t/accessing-xdg-config-home-of-the-host/9780/10)

We can handle the common case by just hard-coding `~/.config`:
```suggestion
    - $HOME/.minikube
    - $HOME/.config/k3d
```

Comment (dholbach) [reply snap/snapcraft.yaml]: Thanks a lot for digging into this!

**Comment by dholbach** [comment]
Thanks for the edit @stealthybox - this is ready to go now. :heavy_check_mark:

**Comment by dholbach** [comment]
This had to go through the store review process, but is live in `edge` now:

```
Track    Arch    Channel    Version          Revision
latest   amd64   stable     1.14.2           30
                 candidate  ^                ^
                 beta       ^                ^
                 edge       1.14.2+bb576fad  35
```

**Pull request merged**